src/sass/style.scss

```diff
@@ -30,3 +30,7 @@ a.not-underline {
     text-decoration: none;
   }
 }
+.carousel-item.active, .carousel-item-next, .carousel-item-prev {
```

Comment:
I can't see elements with `carousel-item-next` or `carousel-item-prev` class. I assume we don't need to style them.